engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -130,6 +158,25 @@ public void transform(final String sourceMimetype, final String targetMimetype,
         {
             pageLimit = parseInt(stringPageLimit, PAGE_LIMIT);
         }
+        String pdfFont = transformOptions.get(PDF_FONT);
+        if (pdfFont == null || pdfFont.isBlank())
+        {
+            pdfFont = pdfBoxDefaultFont;
+        }
+        String pdfFontSize = transformOptions.get(PDF_FONT_SIZE);
+        Integer fontSize = null;
+        if (pdfFontSize != null && !pdfFontSize.isBlank())
+        {
+            try
+            {
+                fontSize = parseInt(pdfFontSize, PDF_FONT_SIZE);
+            }
+            catch (Exception e)
+            {
+                fontSize = DEFAULT_FONT_SIZE;
+                logger.error("Error parsing font size {}, going to set it as {} : {}", pdfFontSize, fontSize, e.getMessage());

```

Comment:
I think you should use standard slf4j exception logging for this:
```suggestion
                logger.error("Error parsing font size {}, going to set it as {}", pdfFontSize, fontSize, e);

```

Reply:
Changed how exceptions are printed according to slf4j pattern.